framework/helpers/BaseHtml.php

```diff
@@ -1157,6 +1157,18 @@ public static function activeInput($type, $model, $attribute, $options = [])
      */
     public static function activeTextInput($model, $attribute, $options = [])
     {
+        if (!isset($options['maxlength'])) {
```

Comment:
Doesn't look right to me. Option isn't meant to be set by user, correct? Either the check should be `isset` without `!` or the whole block should be removed.